elide-swagger/src/main/java/com/yahoo/elide/swagger/SwaggerBuilder.java

```diff
@@ -767,12 +767,10 @@ public Swagger build() {
         for (Type<?> clazz : allClasses) {
             if (clazz instanceof ClassType) {
                 models.putAll(converters.readAll(((ClassType) clazz).getCls()));
-            } else if (clazz instanceof Type) {
+            } else {
```

Comment:
Not sure about this removal.  The type here is the elide Type - not the superclass of Class.

Reply:
`Type<?> clazz` the expression is always true.